doc/onboarding.md

```diff
@@ -14,9 +14,8 @@ onboarding session.
 
 ## Fifteen minutes before the onboarding session
 
-* Prior to the onboarding session, add the new Collaborator to
-  [the Collaborators team](https://github.com/orgs/nodejs/teams/collaborators),
-  and to [the Members team](https://github.com/orgs/nodejs/teams/members) if
+* Prior to the onboarding session, add the new Collaborator to the
```

Comment:
Looks like there's an extra `the` at the end of this line (or an extra one at the beginning of the next line).